BeeKit/Managers/CurrentUserManager.swift

```diff
@@ -82,6 +83,14 @@ public actor CurrentUserManager {
         try! context.save()
     }
 
+    private nonisolated func cleanUpUserDefaults() {
+        for key in CurrentUserManager.allKeys {
+            userDefaults.removeObject(forKey: key)
```

Comment:
though this removes not just elements of user but also the access token; not sure if that was intended

Reply:
Not strictly intended but I think fine - the access token should be stored in the keychain now.